OutOfSchool/OutOfSchool.AuthServer.Tests/Services/CommonMinistryAdminServiceTests.cs

```diff
@@ -21,35 +24,87 @@
 
 namespace OutOfSchool.AuthCommon.Services.Tests;
 
-[TestFixture()]
+[TestFixture]
 public class CommonMinistryAdminServiceTests
 {
-    private ICommonMinistryAdminService<AreaAdminBaseDto> areaCommonMinistryAdminService;
+    private readonly Mock<IEmailSenderService> fakeEmailSender;
+    private readonly Mock<IMapper> fakeMapper;
+    private readonly Mock<ILogger<CommonMinistryAdminService<long, AreaAdmin, AreaAdminBaseDto, AreaAdminRepository>>> fakeLogger;
     private AreaAdminRepository areaAdminRepository;
+    private readonly Mock<UserManager<User>> fakeUserManager;
+    private OutOfSchoolDbContext context;
+    private readonly Mock<IRazorViewToStringRenderer> fakeRenderer;
+    private readonly Mock<IStringLocalizer<SharedResource>> fakeLocalizer;
+    private readonly Mock<IUrlHelper> fakeUrlHelper;
+
+    private ICommonMinistryAdminService<AreaAdminBaseDto> areaCommonMinistryAdminService;
+
+    public CommonMinistryAdminServiceTests()
+    {
+        fakeEmailSender = new Mock<IEmailSenderService>();
+        fakeMapper = new Mock<IMapper>();
+        fakeLogger = new Mock<ILogger<CommonMinistryAdminService<long, AreaAdmin, AreaAdminBaseDto, AreaAdminRepository>>>();
+        fakeRenderer = new Mock<IRazorViewToStringRenderer>();
+        fakeLocalizer = new Mock<IStringLocalizer<SharedResource>>();
+        fakeUrlHelper = new Mock<IUrlHelper>();
+        fakeUserManager = new Mock<UserManager<User>>(
+             new Mock<IUserStore<User>>().Object,
+             new Mock<IOptions<IdentityOptions>>().Object,
+             new Mock<IPasswordHasher<User>>().Object,
+             new IUserValidator<User>[0],
+             new IPasswordValidator<User>[0],
+             new Mock<ILookupNormalizer>().Object,
+             new Mock<IdentityErrorDescriber>().Object,
+             new Mock<IServiceProvider>().Object,
+             new Mock<ILogger<UserManager<User>>>().Object);
```

Comment:
(Test design) By using mocks created in constructor you are introducing state shared between tests. Slightly better performance, worse side-effects. It's common approach to not do this. So, please consider moving all initialization in `SetUp()`. 
Additionally consider removing private fields that are not used anywhere except `SetUp()` (like `fakeEmailSender`, etc.).